static/js/popovers.js

```diff
@@ -72,10 +72,12 @@ function init_email_clipboard() {
         if (this.clientWidth < this.scrollWidth) {
             const email_el = $(this);
             const copy_email_icon = email_el.find('i');
-            copy_email_icon.removeClass('hide_copy_icon');
 
-            const copy_email_clipboard = new ClipboardJS(copy_email_icon[0]);
-            copy_email_clipboard.on('success', copy_email_handler);
+            if (copy_email_icon[0]) {
```

Comment:
Can you add a comment with a quick note about what the meaning is in cases where `copy_email_icon` might not be present?  I think that will help reading this code later understand it.

Reply:
comment was added